### ISSUE TITLE
Commented forms aren't grayed out in deps.edn

### ISSUE THREAD

**Issue by fctorial** [cursive-ide/cursive#2398]
The form prefixed with `#_` should be grayed out.

https://prnt.sc/tg3p4b

**Comment by imrekoszo** [comment]
This appears to be fixed in `1.12.2-eap4-2021.3`: 
![image](https://user-images.githubusercontent.com/1116295/154070683-30afdc21-b53d-45ce-856d-0673abeaa2d4.png)

```
IntelliJ IDEA 2021.3.2 (Community Edition)
Build #IC-213.6777.52, built on January 28, 2022
Runtime version: 11.0.13+7-b1751.25 x86_64
VM: OpenJDK 64-Bit Server VM by JetBrains s.r.o.
macOS 12.2.1
GC: G1 Young Generation, G1 Old Generation
Memory: 4096M
Cores: 8
Registry:
    scala.erase.compiler.process.jdk.once=false

Non-Bundled Plugins:
    org.intellij.plugins.hcl (0.7.15)
    net.seesharpsoft.intellij.plugins.csv (2.18.2)
    name.kropp.intellij.makefile (213.5744.224)
    de.thomasrosenau.diffplugin (2.1.8-imrekoszo)
    com.namespace.sort.namespace-sort-plugin (1.7-imrekoszo-2021-3-linebreak)
    com.jetbrains.plugins.ini4idea (213.5744.190)
    org.jetbrains.kotlin (213-1.6.10-release-961-IJ6777.52)
    org.jetbrains.idea.grammar (2021.1.2)
    gherkin (213.6461.19)
    dev.monogon.cuelang (0.8.3)
    StringToolsPlugin (4.0)
    String Manipulation (8.27.203.5981.1)
    Key Promoter X (2021.3)
    mobi.hsz.idea.gitignore (4.3.0)
    com.cursiveclojure.cursive (1.12.2-eap4-2021.3)
    com.github.shinichy.integrant (0.1.1)
    com.github.brcosta.cljstuffplugin (0.5.4)
    EDN-JSON Converter (0.0.5)
    org.intellij.scala (2021.3.18)

Kotlin: 213-1.6.10-release-961-IJ6777.52
```

**Comment by cursive-ide** [comment]
Yes, this has been fixed for a while, thanks.
